src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.csproj

```diff
@@ -1,13 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
Are we sure dropping `netstandard2.1` here is correct? Looks like @maryamariyan added it during getting this project moved over to this repo.  [https://github.com/dotnet/runtime/commit/222043719d772ca9abea2131dce6412ef1b2a159](https://github.com/dotnet/runtime/commit/222043719d772ca9abea2131dce6412ef1b2a159)

Reply:
The netstandard2.1 reference isn't needed as netstandard2.0 fully satisfies the contract. It's not entirely clear to me why that configuration was added when bringing over the project but it doesn't serve any purpose, especially now that we are adding a .NETCoreApp configuration.